internal/kafka/internal/config/aws_config.go

```diff
@@ -0,0 +1,101 @@
+package config
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared"
+	"github.com/spf13/pflag"
+)
+
+type AWSConfig struct {
+	Route53                     awsRoute53Config
+	SecretManager               awsSecretManagerConfig
+	ConfigForOSDClusterCreation awsConfigForOSDClusterCreation
+}
+
+type awsSecretManagerConfig struct {
+	AccessKey               string
+	AccessKeyFilePath       string
+	SecretAccessKey         string
+	SecretAccessKeyFilePath string
+	SecretPrefix            string
+	Region                  string
+}
+
+type awsConfigForOSDClusterCreation struct {
+	AccountID               string
+	AccountIDFilePath       string
+	AccessKey               string
+	AccessKeyFilePath       string
+	SecretAccessKey         string
+	SecretAccessKeyFilePath string
+}
+
+type awsRoute53Config struct {
+	AccessKey               string
+	AccessKeyFilePath       string
+	SecretAccessKey         string
+	SecretAccessKeyFilePath string
+}
+
+func NewAWSConfig() *AWSConfig {
+	return &AWSConfig{
+		ConfigForOSDClusterCreation: awsConfigForOSDClusterCreation{
+			AccountIDFilePath:       "secrets/aws.accountid",
+			AccessKeyFilePath:       "secrets/aws.accesskey",
+			SecretAccessKeyFilePath: "secrets/aws.secretaccesskey",
+		},
+		Route53: awsRoute53Config{
+			AccessKeyFilePath:       "secrets/aws.route53accesskey",
+			SecretAccessKeyFilePath: "secrets/aws.route53secretaccesskey",
+		},
+		SecretManager: awsSecretManagerConfig{
+			AccessKeyFilePath:       "secrets/aws-secret-manager/aws_access_key_id",
```

Comment:
Do the file paths need to be exported? Can we make them unexported?
In that way we hide this from consumers of the types outside the package